package/gluon-luci-node-role/files/usr/lib/lua/luci/controller/admin/noderole.lua

```diff
@@ -1,5 +1,5 @@
 module("luci.controller.admin.noderole", package.seeall)
 
 function index()
-	entry({"admin", "noderole"}, cbi("admin/noderole"), "Verwendungszweck", 20)
+	entry({"admin", "noderole"}, cbi("admin/noderole"), "Purpose", 20)
```

Comment:
Hmm, I don't really like the term "purpose" here. Can we just keep calling this "Node role"?